source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -2353,7 +2443,7 @@ def test_delay_pass_midnight_towards_previous_day(self):
 
         # /journeys from C to A with departure at 20120614T223000 and arrival at 20120615T080102
         journey_query = (
-            "journeys?from={f}&to={to}&data_freshness=realtime&"
+            "journeys?from={f}&to={to}&data_freshness=realtime&max_duration_to_pt=0&"
```

Comment:
max_duration_to_pt=0 ?

Reply:
This parameter is to avoid a section on walk.